### DISCUSSION TITLE
my problem of bands_mat.x?

### DISCUSSION THREAD

**Discussion by nvhungUCL** [nguyen-group/QERaman#14] [Q&A] [answered]
Dear developers,

I am very interested in your codes and learning them.
However, I met a small problem and I can not understand why.
Could you explain me?

In particular, I am testing calculations for graphene using your input files in the directory "examples". I had a problem in calculations of bands_mat.x as you can see bellow. Attached comments are in the file bands.out.
It told me that the code can not read the files "graphene.wfc..."

Could you suggest me how to solve this problem?

Thank you very much. 
Viet Hung Nguyen

---------------------------------------------------------------

...
     e(  4 -  4) =     -7.72079  eV     1   --> A              
     e(  5 -  5) =     -1.07932  eV     1   --> A              
     e(  6 -  6) =     -0.11518  eV     1   --> A              
     e(  7 -  7) =      0.82140  eV     1   --> A              
     e(  8 -  8) =      1.25186  eV     1   --> A              

 **************************************************************************

 **************************************************************************

                    xk=(  -0.01667,  -0.02887,   0.00000  )

     Band symmetry, C_1 (1)     point group:

     e(  1 -  1) =    -23.84778  eV     1   --> A              
     e(  2 -  2) =    -12.16397  eV     1   --> A              
     e(  3 -  3) =     -7.69964  eV     1   --> A              
     e(  4 -  4) =     -7.66388  eV     1   --> A              
     e(  5 -  5) =     -1.11466  eV     1   --> A              
     e(  6 -  6) =     -0.15113  eV     1   --> A              
     e(  7 -  7) =      0.78513  eV     1   --> A              
     e(  8 -  8) =      1.21578  eV     1   --> A              

 **************************************************************************
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
     Error in routine davcio (10):
     error reading file "/gpfs/home/acad/ucl-modl/vhnguyen/softwares/QuanESPRE7.2/QERaman/examples/graphene/./tmp/graphene.wfc45"
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

     stopping ...

 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
     Error in routine davcio (10):
     error reading file "/gpfs/home/acad/ucl-modl/vhnguyen/softwares/QuanESPRE7.2/QERaman/examples/graphene/./tmp/graphene.wfc47"
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

     stopping ...

 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
     Error in routine davcio (10):
 ...

**Comment by nguyen-group** [comment]
Hi Viet Hung Nguyen,

Could you check "JOB DONE." in the scf.out to see if the scf step was done or not?
If is done, can you check to run bands.in with bands.x to check if the QE can work or not?

Best all,
Nguyen Tuan Hung

**Reply by nvhungUCL** [reply]
Dear Tuan-Hung,

Thank for your suggestions.
I checked it. It is "JOB DONE" in scf.out but indeed, bands.x did not work.
I found the same error as with bands_mat.x.
Could you suggest me what I should do?

Thank you very much,
Viet-Hung Nguyen

**Reply by nguyen-group** [reply]
Hi, 

If bands.x does work, that means installed QE might have the problem.
You might need to check the permissions of the tmp folder or reinstall QE.

Best all,
Nguyen Tuan Hung

**Reply by nvhungUCL** [reply]
Thank you very much,

You are totally correct! The problem was indeed of my QE version.
After I reinstalled my QE, QERaman worked well for the example of graphene as presented in your CPC article.

Thank you again for your helps.
Viet-Hung Nguyen

**Reply by nguyen-group** [reply, marked as answer]
you are welcome.
